create.py

```diff
@@ -80,4 +84,62 @@ def logout():
         return render_template("reg.html", var1 = var1)
     except:
         var1 = "You must first log in to logout"
-        return render_template("reg.html",var1 = var1)
\ No newline at end of file
+        return render_template("reg.html",var1 = var1)
+@app.route('/books/<id>',methods=['POST','GET'])
+def books(id):
+    print(id)
```

Comment:
check if the user is logged in or not before accepting any review.

Reply:
updated user is login or not in create.py